src/java/org/apache/cassandra/net/SensorsCustomParams.java

```diff
@@ -36,13 +36,23 @@ public final class SensorsCustomParams
      * The total read bytes value for a given keyspace and table, across all requests. This is a monotonically increasing value.
      */
     public static final String READ_BYTES_TABLE = "READ_BYTES_TABLE";
+    /**
+     * The per-request write bytes value for a given keyspace and table.
+     * To support batch writer, the keyspace and table are encoded in the following format: WRITE_BYTES_REQUEST.<keyspace>_<table>
```

Comment:
I think encoding the keyspace is redundant as a mutation map 1:1 with ks, but will look more into it especially with write batches across keyspaces (@sbtourist if you have any thoughts?)

Reply:
So ks is redundant - coordinator writer response handlers would map 1:1:1 with replica plans and ks. Removing.